config.example.yaml

```diff
@@ -53,6 +53,7 @@ show_image_date: false # true or false
 image_date_format: YYYY-MM-DD
 show_image_exif: false
 show_image_location: false
+hide_countries: []
```

Comment:
I don't believe this is valid yaml, but either way I would probably document this like:

```yaml
hide_countries:
  - "country to hide"
```

Or something of that nature.

Reply:
I believe it is [valid yaml](https://www.yaml.info/learn/flowstyle.html), but happy to change for readability

Reply:
I do apologise. That will teach me to do PR reviews just before going to sleep!